### DISCUSSION TITLE
Use Postgres for database

### DISCUSSION THREAD

**Discussion by stephenWanjala** [be-great/Automsg#6] [Ideas]
Hey @be-great  Should I set The project to Use PostgreSQL?
And Also maybe help in setting up docker files

**Comment by be-great** [comment]
@stephenWanjala  we voted to use Mysql.

**Reply by stephenWanjala** [reply]
Okay ,Then we set up the project to use Mysql instead of sqlite

**Reply by be-great** [reply]
Yes

**Comment by be-great** [comment]
What part you have problem with in dockerfile. The docker container when you run it, it already run mysql server on 3306 port

**Reply by be-great** [reply]
![Screenshot from 2024-01-26 13-36-44.png](https://github.com/be-great/Automsg/assets/78013422/e7e6f044-3d0d-4152-9f17-c2ad4eed6be7)

![Screenshot from 2024-01-26 13-36-00.png](https://github.com/be-great/Automsg/assets/78013422/ff026ccf-ae60-4cec-aa2e-c32924ed0672)

**Reply by be-great** [reply]
@stephenWanjala  you should run the docker container then change the settings.py in the Django app
![Screenshot_20240129_110347_com.openai.chatgpt_edit_34954440856644.jpg](https://github.com/be-great/Automsg/assets/78013422/8f38f954-cb58-4590-8100-fb531b64628c)
Since this is the setting on the "docker_compose.yml" 
![Screenshot_20240129_110758_com.github.android_edit_35205847298793.jpg](https://github.com/be-great/Automsg/assets/78013422/a4d9ce61-55be-438f-a189-fd1aac99d555)

**Reply by stephenWanjala** [reply]
I hadn't checked out the updated conf
But still the backend is using SQL lite

**Reply by be-great** [reply]
ok
